comp/core/status/statusimpl/status_api_endpoints.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package statusimpl implements the status component interface
+package statusimpl
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Comment:
Should we use the log component? It seems that on line 67 we should be use `s.log*` 😄

Reply:
You're totally right ! Done in 5020fd93fb0946db9dcf7cee69c71d80ffeb5b2f.